### ISSUE TITLE
Question: Can I mix codecs?

### ISSUE THREAD

**Issue by colleenkhenry** [google/shaka-packager#571]
I'm curious, is it possible with Packager to mix codecs in the same live stream. For example, AV1 on lower levels and VP9 on higher ones?

**Comment by kqyang** [comment]
Yes, you can. 

You won't be able to adapt between the two set of streams though, e.g. they are put in two different AdaptationSets if you are using DASH.

Does that answer your question?

**Comment by jakubvojacek** [comment]
Hello @kqyang,

is this a limitation of shaka packager or of a dash? We too would like to have ABR working in between two different between codecs.

Thank you

**Comment by kqyang** [comment]
@jakubvojacek It is more of the limitation of the platform as most platforms do not support adaptive switching between different codecs. What codecs are you thinking of? There was a request some time ago to support adaptive switching between FLAC and Opus: #542.

**Comment by jakubvojacek** [comment]
@kqyang we were hoping to adapt in between h264 and hevc. h264 in lower bitrates for mobile platforms and highest bitrate in hevc for the settopboxes.

**Comment by jakubvojacek** [comment]
@kqyang I have found https://github.com/google/ExoPlayer/issues/2431#issue-206362657 which is exactly what we need. We have two adaptation sets, one in hevc and h264. I tried adding 
`<SupplementalProperty schemeIdUri="urn:mpeg:dash:adaptation-set-switching:2016" value="1"/>
` to the hevc adaptation and `<SupplementalProperty schemeIdUri="urn:mpeg:dash:adaptation-set-switching:2016" value="2"/>` to the h264 adaptation and it worked flawlessly - the exoplayer was able to adapt in between codecs. 

Is there a way to tell the shaka packager to add the `SupplementalProperty`? You are already generating it for adaptation sets that are having same codec but different kid, we need it to be available across codecs too.

Thank you
